src/HotReload/FSEventParser.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace SwooleTW\Http\HotReload;
+
+use Carbon\Carbon;
+
+/**
+ * Class FSEventParser
+ */
+class FSEventParser
+{
+    protected const REGEX = '/^([\S+]{3}\s+[\S+]{3}\s+[\d+]{2}\s+[\d+]{2}:[\d+]{2}:{0,2}:[\d+]{2}:{0,2}\s+[\d+]{0,4})\s+(\/[\S+]*)\s+([\S+*\s+]*)/mi';
```

Comment:
I found the reason that `FSEventParser::toEvent($event)` is always `null` in my environment. I'm using OSX and `fswatch 1.14.0`. Here's what I received for the fswatch event:

```
Mon Jan  7 23:30:28 2019 /Users/Albert/Projects/laravel-swoole-demo/vendor/swooletw/laravel-swoole/src/HotReload/FSProcess.php PlatformSpecific Updated IsFile
```

I haven't tested it on other platforms. But it works for my after changing the regex to:

```
^([\S+]{3}\s+[\S+]{3}\s+[\d+]{1,2}\s+...
```

Reply:
Fixed

Reply:
Hi @fractalzombie ,

Thank you so much ^^